api/v1alpha1/common_types.go

```diff
@@ -5,3 +5,11 @@ type NamespacedName struct {
 	Name      string `json:"name"`
 	Namespace string `json:"namespace"`
 }
+
+// SlaSpec defines what is the percentage
+type SlaSpec struct {
+	// Percentile is the fraction of the time that resource needs to be availble at
+	Percentile int `json:"percentile"`
+	// Precision is the precision the percentile will be (default to the length of the percentile + 1)
+	Precision int `json:"precision,omitempty"`
```

Comment:
so with having to integers, to define a target availability of 99.5 I would need to set the Percentile to 995 and the precision to 1000? That sounds a bit complicated to me. Why not just use `TargetAvailabilityPercentage` as floating point value, so the precision is clear from the name?

Reply:
Changed it back to string and an enum with "percent" or "percentile"

Reply:
@NautiluX floating point is not supported, so could'nt do that.. the only good thing is having a string and parsing it :)